codecell/calculator
# #Calculator_project_Milestone_1

**What does this PR do?**
This is for TSE review of milestone implementation.

**Tasks completed**
-  basic setup and deployment of create-react-app boilerplate code.
- Added prop-type and big.js packages.

**Live version**
[calculatepro.herokuapp](https://calculatepro.herokuapp.com/)

**How can this be manually tested?**
- clone this repo
- run `yarn install` OR `npm install` to install project dependencies
- run `yarn start` OR `npm start` to serve a development build on localhost:://3000
- You should be greeted with the basic react.js welcome page on visiting http://localhost:3000/ signifying success.
